app/src/main/java/cm/aptoide/pt/app/view/AppViewPresenter.java

```diff
@@ -775,7 +812,11 @@ private void handleInstallButtonClick() {
                                   appViewModel.getPackageName(), appViewModel.getDeveloper()
                                       .getName(), action.toString()))
                               .doOnCompleted(() -> showRecommendsDialog(account.isLoggedIn(),
-                                  appViewModel.getPackageName())));
+                                  appViewModel.getPackageName()))
+                              .toSingleDefault(true)
+                              .delay(3, TimeUnit.SECONDS)
```

Comment:
Was this discussed with Felix? time based approach seems to be really bad and inconsistent in terms of ux. Also it looks like we are throwing this on top of the share dialog.